### PR TITLE
chore: Checkout [check-run-reporter](https://www.check-run-reporter.com/docs)'s reporting.

### PR DESCRIPTION
**Pull request by randytarampi**
Just opting for the basic JUnit ESLint output for now, but maybe we'll go down a different reporting route, so let's cross that bridge when we get there...

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -133,13 +135,23 @@
     "clean": "./bin/clean.sh",
     "cover": "yarn run cover:actual; yarn run cover:collect",
     "cover:actual": "rm -rf .nyc_output/ coverage/ && lerna run cover --no-bail --concurrency $(node -p \"require(\\\"os\\\").cpus().length\")",
-    "cover:collect": "mkdir -p .nyc_output/ coverage/ && lerna exec -- \"if [ -d \"./.nyc_output\" ]; then cp -r .nyc_output ../..; fi;\" && nyc report --reporter=lcov",
+    "cover:actual:collect:lcov": "mkdir -p .nyc_output/ coverage/ && lerna exec -- \"if [ -d \"./.nyc_output\" ]; then cp -r .nyc_output ../..; fi;\" && nyc report --reporter=lcov",
+    "cover:collect": "yarn run test:collect; yarn run cover:actual:collect:lcov",
     "coveralls": "cat coverage/lcov.info | coveralls",
+    "check-run-reporter:actual": "curl \"https://api.check-run-reporter.com/api/v1/submissions\" --user $CHECK_RUN_REPORTER_TOKEN -X POST",
+    "check-run-reporter:test": "yarn run check-run-reporter:actual -- -F report=@\"./test-results.xml\" -F label=\"test\" -F sha=\"COMMIT_SHA\" -F root=\"$(pwd)\"",
```
Comment (ianwremmel) [inline package.json]: Hi! I'm the CEO over at Check Run Reporter and I noticed some errors in the logs which are preventing you from seeing test results. I'll have a few documentation and code fixes coming along soonish, but in the meantime, you'll want to use the travis commit sha variable here:

```suggestion
    "check-run-reporter:test": "yarn run check-run-reporter:actual -- -F report=@\"./test-results.xml\" -F label=\"test\" -F sha=\"$TRAVIS_COMMIT\" -F root=\"$(pwd)\"",
```

I'm super excited to see folks using CRR so if you run into any other issues, feel free to reach out and I'll do what I can. Hope this helps!

File: package.json [hunk 2]
```diff
@@ -133,13 +135,23 @@
...
+    "check-run-reporter:pretest": "yarn run check-run-reporter:actual -- -F report=@\"./pretest-results.xml\" -F label=\"pretest\" -F sha=\"COMMIT_SHA\" -F root=\"$(pwd)\"",
```
Comment (ianwremmel) [inline package.json]: Same thing here:

```suggestion
    "check-run-reporter:pretest": "yarn run check-run-reporter:actual -- -F report=@\"./pretest-results.xml\" -F label=\"pretest\" -F sha=\"TRAVIS_COMMIT\" -F root=\"$(pwd)\"",
```

**Comment by ianwremmel** [comment]
JUnit is a fickle beast :)

It looks like ESLint changed their JUnit report format since I implemented its parser. You'll get better results for ESLint using the json-with-metadata outputer. Do you mind if I add your failed test reports to the test suite in order to fix the parse errors you're seeing?

**Comment by randytarampi** [comment]
Haha. Go for it.

**Comment by ianwremmel** [comment]
Fixes for both eslint and mocha JUnit parsing have been deployed. Hopefully that'll get you unblocked. Thanks for your patience!

**Comment by ianwremmel** [comment]
Looks like `COMMIT_SHA` without a leading `$` snuck back into the code somewhere. There'll be a change shortly that sends that error back to you instead of forcing me to reach out :)

**Pull request merged**